plugins/scaffolder/src/components/TemplatePage/TemplatePage.tsx

```diff
@@ -65,7 +65,8 @@ const OWNER_REPO_SCHEMA = {
       format: 'GitHub user or org / Repo name',
       type: 'string' as const,
       title: 'Store path',
-      description: 'GitHub store path in org/repo format',
+      description:
+        'GitHub store path in org/repo format. Scaffolding under personal accounts is currently not supported.',
```

Comment:
This is about to be fixed by https://github.com/spotify/backstage/pull/2433, so perhaps skip this one?

Reply:
Didn't see that. Done.